contrib/macdeploy/background.svg

```diff
@@ -1,34 +1,33 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
-<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="1000pt" height="640pt" viewBox="0 0 1000 640" preserveAspectRatio="xMidYMid meet">
-	<!-- kate: space-indent off;
-	Copyright (c) 2015 The Bitcoin Core developers
-	Distributed under the MIT software license, see the accompanying
-	file COPYING or http://www.opensource.org/licenses/mit-license.php.
-	-->
-	<style type="text/css"><![CDATA[
-		text {
-			font-family: "Tuffy";
-			font-size: 86px;
-			fill: gray;
-			text-anchor: middle;
-		}
-	]]></style>
-	<defs>
-		<linearGradient id="gradient" x1="0%" y1="0%" x2="100%" y2="100%">
-			<stop offset="0%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
-			<stop offset="33%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
-			<stop offset="80%" style="stop-color:rgb(205,205,205);stop-opacity:1" />
-			<stop offset="100%" style="stop-color:rgb(204,204,204);stop-opacity:1" />
-		</linearGradient>
-	</defs>
-	<rect width="1000" height="640" style="fill:url(#gradient);stroke-width:0" />
-	<g transform="translate(500,0) scale(0.9, 1)">
-		<text x="0" y="114">PACKAGE_NAME</text>
-	</g>
-	<g transform="translate(0.000000,640.000000) scale(0.100000,-0.100000)"
-	fill="#000000" stroke="none">
-		<path d="M4995 3705 c-24 -23 -25 -29 -25 -165 l0 -140 -306 0 -306 0 -29 -29 c-29 -29 -29 -31 -29 -141 0 -110 0 -112 29 -141 l29 -29 306 0 306 0 0 -140 c0 -136 1 -142 25 -165 16 -17 35 -25 57 -25 29 0 72 32 306 226 180 149 274 233 278 250 13 53 -2 70 -278 299 -235 194 -277 225 -306 225 -22 0 -41 -8 -57 -25z" fixlter="url(#glow)"/>
-	</g>
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 640" preserveAspectRatio="xMidYMid meet">
+  <!-- kate: space-indent off;
+  Copyright (c) 2015 The Bitcoin Core developers
+  Distributed under the MIT software license, see the accompanying
+  file COPYING or http://www.opensource.org/licenses/mit-license.php.
+  -->
+  <style type="text/css"><![CDATA[
+    text {
+      font-family: "Tuffy";
+      font-size: 86px;
+      fill: gray;
+      text-anchor: middle;
+    }
+  ]]></style>
+  <defs>
+    <linearGradient id="gradient" x1="0%" y1="0%" x2="100%" y2="100%">
+      <stop offset="0%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
+      <stop offset="33%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
+      <stop offset="80%" style="stop-color:rgb(205,205,205);stop-opacity:1" />
+      <stop offset="100%" style="stop-color:rgb(204,204,204);stop-opacity:1" />
+    </linearGradient>
+  </defs>
+<rect width="1000" height="640" style="fill:url(#gradient);stroke-width:0" />
+  <g transform="translate(500,0) scale(1.0, 1.0)">
```

Comment:
Why this change?

Reply:
@promag - according to documentation the scaling/aspect ratio is taken care of in the 
preserveAspectRatio flag in 

LINE:4 
```html
<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="1000pt" height="640pt" viewBox="0 0 1000 640" preserveAspectRatio="xMidYMid meet">
```
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio

```text
Meet or slice reference
The meet or slice reference is optional and, if provided, must be one of the following keywords:
meet (the default) - Scale the graphic such that:
aspect ratio is preserved
the entire viewBox is visible within the viewport
the viewBox is scaled up as much as possible, while still meeting the other criteria
```

Reply:
Also https://developer.gnome.org/rsvg/stable/recommendations-assets.html

Reply:
> Why this change?

Here is the result now with the previous scale settings
![Screen Shot 2019-10-27 at 4 53 02 PM](https://user-images.githubusercontent.com/152159/67641696-7de17f80-f8da-11e9-8326-824391376933.png)

Reply:
@promag - now that I have a better understanding of how to configure this - it is now more of an aesthetic choice than dealing with a kerning issue. In fact I plan to rework the whole presentation in a later PR - I just want to fix the glaring issue now - then focus on a better presentation later.